cns/restserver/v2/server.go

```diff
@@ -0,0 +1,61 @@
+package v2
+
+import (
+	"net/http"
+
+	"github.com/Azure/azure-container-networking/cns"
+	"github.com/Azure/azure-container-networking/cns/restserver"
+	"github.com/labstack/echo/v4"
+	"go.uber.org/zap"
+)
+
+type Server struct {
+	*restserver.HTTPRestService
```

Comment:
you're getting 
`s.RequestIPConfigHandler undefined (type *Server has no field or method RequestIPConfigHandler) (typecheck)`
because this is embedded as a reference type. embed it as a value type and that method will be available on a `*Server`